contrib/managed-azure/delete_hosted_cluster.sh

```
@@ -29,17 +29,16 @@ CUSTOMER_VNET_RG_NAME="${PREFIX}-customer-vnet-rg"
 CUSTOMER_NSG_RG_NAME="${PREFIX}-customer-nsg-rg"
 CLUSTER_NAME="${PREFIX}-hc"
 
-# Delete the hosted cluster using hypershift destroy command
+# Delete the hosted cluster using hypershift destroy command.
+# If the HostedCluster object doesn't exist (e.g. creation failed before it was applied),
+# hypershift destroy will fail. We log the error but continue so Azure resource groups are
+# still cleaned up below.
 ${HYPERSHIFT_BINARY_PATH}/hypershift destroy cluster azure \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)